test/test_functionalization.py

```diff
@@ -174,6 +174,17 @@ def f(x):
         [1., 1.],
         [1., 1.]]))""")
 
+    # Some ops that are mutable are neither inplace nor out= ops.
+    # They also need special handling.
+    def test_mutable_op_not_inplace_or_other(self):
+        def f(x):
+            return torch._fused_moving_avg_obs_fq_helper(x, x, x, x, x, x, x, 1.0, 1, 1, 0)
+
+        logs = self.get_logs(f, torch.ones(1))
+        self.assertExpectedInline('\n'.join(logs), """\
+$0 = input('input')
+$1, $2, $3, $4, $5, $6 = torch._ops.aten._fused_moving_avg_obs_fq_helper.functional($0, $0, $0, $0, $0, $0, $0, 1.0, 1, 1, 0)""")
```

Comment:
cc @anijain2305, this is what the graph will look like when you functionalize `_fused_moving_avg_obs_fq_helper` (all of the mutable args in the original op become outputs)